osquery/tables/other/yara.cpp

```diff
@@ -147,35 +146,16 @@ QueryData genYara(QueryContext& context) {
       }
 
       VLOG(1) << "Scanning with group: " << group;
-      status = doYARAScan(rules[group],
-                          path_pair.first.c_str(),
-                          path_pair.second,
-                          results,
-                          group,
-                          "");
+      auto status = doYARAScan(rules[group],
+                               path_pair.first.c_str(),
+                               path_pair.second,
+                               results,
+                               group,
+                               group);
```

Comment:
The group is used as either the sig_group or sigfile when emitted row data. This is because sigfiles are read once and their path is converted to the cache key = group name.